### ISSUE TITLE
500 Error on .spacewiki/navigation-list

### ISSUE THREAD

**Issue by kiery** [spacewiki/spacewiki#5]
If you go from docs > special pages > navigation list a 500 error is returned.

**Comment by tdfischer** [comment]
Server logs:

```
Dec 01 21:29:37 app1 gunicorn[1177]: ERROR:spacewiki.app:Exception on /spacewiki/navigation-list [GET]
Dec 01 21:29:37 app1 gunicorn[1177]: Traceback (most recent call last):
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
Dec 01 21:29:37 app1 gunicorn[1177]:     response = self.full_dispatch_request()
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
Dec 01 21:29:37 app1 gunicorn[1177]:     rv = self.handle_user_exception(e)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
Dec 01 21:29:37 app1 gunicorn[1177]:     reraise(exc_type, exc_value, tb)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
Dec 01 21:29:37 app1 gunicorn[1177]:     rv = self.dispatch_request()
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
Dec 01 21:29:37 app1 gunicorn[1177]:     return self.view_functions[rule.endpoint](**req.view_args)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/app/spacewiki/pages.py", line 69, in view
Dec 01 21:29:37 app1 gunicorn[1177]:     missingIndex=missingIndex)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/flask/templating.py", line 134, in render_template
Dec 01 21:29:37 app1 gunicorn[1177]:     context, ctx.app)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/flask/templating.py", line 116, in _render
Dec 01 21:29:37 app1 gunicorn[1177]:     rv = template.render(context)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
Dec 01 21:29:37 app1 gunicorn[1177]:     return self.environment.handle_exception(exc_info, True)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
Dec 01 21:29:37 app1 gunicorn[1177]:     reraise(exc_type, exc_value, tb)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/app/spacewiki/../templates/page.html", line 1, in top-level template code
Dec 01 21:29:37 app1 gunicorn[1177]:     {% extends "layout.html" %}
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/app/spacewiki/../templates/layout.html", line 59, in top-level template code
Dec 01 21:29:37 app1 gunicorn[1177]:     {% block tools %}
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/app/spacewiki/../templates/layout.html", line 74, in block "tools"
Dec 01 21:29:37 app1 gunicorn[1177]:     {% block canvas %}
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/app/spacewiki/../templates/layout.html", line 77, in block "canvas"
Dec 01 21:29:37 app1 gunicorn[1177]:     {% block content %}
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/app/spacewiki/../templates/page.html", line 90, in block "content"
Dec 01 21:29:37 app1 gunicorn[1177]:     {% include "_navtree.html" %}
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/app/spacewiki/../templates/_navtree.html", line 7, in top-level template code
Dec 01 21:29:37 app1 gunicorn[1177]:     {% for sibling in page.siblings %}
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/jinja2/environment.py", line 408, in getattr
Dec 01 21:29:37 app1 gunicorn[1177]:     return getattr(obj, attribute)
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/app/spacewiki/model.py", line 197, in siblings
Dec 01 21:29:37 app1 gunicorn[1177]:     parent = Page.select().where(Page.slug == parentSlug)[0]
Dec 01 21:29:37 app1 gunicorn[1177]:   File "/srv/spacewiki/virtualenv/lib/python2.7/site-packages/peewee.py", line 2657, in __getitem__
Dec 01 21:29:37 app1 gunicorn[1177]:     return res._result_cache[value]
Dec 01 21:29:37 app1 gunicorn[1177]: IndexError: list index out of range
```

**Comment by tdfischer** [comment]
Fixed in 6780466f.
